starlette/applications.py

```diff
@@ -1,15 +1,19 @@
+import asyncio
+import inspect
+import typing
+
 from starlette.exceptions import ExceptionMiddleware
 from starlette.lifespan import LifespanHandler
 from starlette.requests import Request
 from starlette.routing import Path, PathPrefix, Router
 from starlette.types import ASGIApp, ASGIInstance, Receive, Scope, Send
 from starlette.websockets import WebSocket
-import asyncio
-import inspect
-import typing
 
 
-def request_response(func):
+Methods = typing.Sequence[str]
```

Comment:
Okay, final thing. Let’s drop `Methods` and just use the `typing.Sequence[str]` annotation directly.

Then I think we’re done on this one!

Reply:
OK!